resources/views/_partials/_delete_modal.blade.php

```diff
@@ -1,6 +1,9 @@
 <form-modal identifier="{{ $identifier }}" :active-modal="activeModal">
     <div class="modal-content" v-cloak>
-        {{ Form::open(['route' => $route, 'method' => 'DELETE']) }}
+        <form action="{{ route($route, $routeParams) }}" method="POST">
```

Comment:
Use argument unpacking instead of the extra variable:

```suggestion
        <form action="{{ route(...$route) }}" method="POST">
```